components/micro-gateway-core/src/main/ballerina/gateway/filters/subscription_filter.bal

```diff
@@ -55,6 +55,23 @@ public type SubscriptionFilter object {
                             APIConfiguration apiConfig = getAPIDetailsFromServiceAnnotation(reflect:
                                 getServiceAnnotations(filterContext.serviceType));
                             if (subscribedAPIList != null){
+                                int l = lengthof subscribedAPIList;
+                                if (l == 0){
```

Comment:
Can we use (lengthof subscribedAPIList == 0) here?

Reply:
this code has revamped with 4d34de4